nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -3247,6 +3247,9 @@ For example, when running in a Docker container or behind a proxy (e.g. localhos
 host[:port] that NiFi is bound to.
 |`nifi.web.proxy.context.path`|A comma separated list of allowed HTTP X-ProxyContextPath, X-Forwarded-Context, or X-Forwarded-Prefix header values to consider. By default, this value is
 blank meaning all requests containing a proxy context path are rejected. Configuring this property would allow requests where the proxy path is contained in this listing.
+|`nifi.web.max.content.size`|The maximum size for regular PUT and POST requests.  The default value is `10 MB`.
```

Comment:
These could use better descriptions.  Why 2 size settings?  When does each get used?  Do they both need to be set?  Should only one be set?

Path to what?  what is it used for?  Why not /tmp?

Reply:
Ok, looking again I understand what path is, sorry.

Reply:
I know you resolved this @alopresto, but this doesn't tell when each large setting should be used, or if they can be used together  or what not.  Still confusing.

Reply:
Sorry, I thought you had reached an understanding of the settings. @natural this is confusing to people, so we should improve the documentation around it.

Reply:
No problem, my confusion was around what "path" meant.  I usually think of this as 'route' and associate path with like a download location.

Reply:
@ottobackwards thanks for the great feedback!

i've gone ahead and simplified this into a single property, `nifi.web.max.content.size`, no paths needed.